actions/bumper/entrypoint.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/sh
+
+# based off https://github.com/anothrNick/github-tag-action
+
+set -eu
+
+# input configuration and defaults
+RELEASE_BRANCH_LIST=${INPUT_RELEASE_BRANCH_LIST:-develop,master}
+DEFAULT_SEMVER_BUMP=${INPUT_SEMVER_BUMP:-minor}
+VERSION_FILE_PATH=${INPUT_VERSION_FILE_PATH:-}
+INITIAL_VERSION=${INPUT_INITIAL_VERSION:-0.0.0}
+SEMVER_WITH_V=${INPUT_SEMVER_WITH_V:-false}
+
+# use comma as input field separator
+PREVIFS=${IFS}
+IFS=','
+
+# check whether we are on a prerelease branch
+CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+PRE_RELEASE=true
+for BRANCH in ${RELEASE_BRANCH_LIST} ; do
+    if expr "${CURRENT_BRANCH}" : "${BRANCH}" 1>/dev/null; then
+        PRE_RELEASE=false
+    fi
+done
+
+# restore previous IFS variable
+IFS=${PREVIFS}
+
+# get all commits since last semantic version
+CURRENT_SEMVER=$(git for-each-ref --sort=-v:refname --count=1 --format '%(refname)' refs/tags/[0-9]*.[0-9]*.[0-9]* refs/tags/v[0-9]*.[0-9]*.[0-9]* | cut -d / -f 3-)
+CURRENT_SEMVER=0.50.0
```

Comment:
What is `0.50.0` and why does it overwrite the previous call to `git`?

Reply:
Whoops, this should be removed, I was using it for testing. Thanks for the catch.